exercises/practice/largest-series-product/example.cpp

```diff
@@ -0,0 +1,36 @@
+#include "example.h"
+#include <numeric>
+#include <algorithm>
+#include <stdexcept>
+#include <regex>
+namespace largest_series_product {
+
+    int largest_product(const std::string& series, int size) {
+        if (size == 0) {
+            return 1;
+        }
+
+        if(size < 0) {
+            throw std::domain_error("span must not be negative");
+        }
+
+        if ((int)series.size() < size) {
+            throw std::domain_error("span must be smaller than string length");
+        }
+
+        if (!std::regex_match(series, std::regex("^\\d*$"))) {
```

Comment:
I don't think you need `^` and `$` because `std::regex_match()` matches the whole sequence.